Documentation/parameters.par

```diff
@@ -16,6 +16,9 @@ ITMStartingtime = 2.0 ! Time when ITM is turned on
 ITMTime = 0.01 !Time duration for which ITM is on
 ITMVelocityscalingfactor = 2 ! Scaling factor
 ITMReflectionType = 1 ! 1 reflects both waves, 2 reflects both waves with constant velocity, 3 reflects only P-waves, 4 reflects only S-waves
+
+numflux = 'godunov' ! The numerical flux near the fault. Values: 'godunov', 'rusanov'
```

Comment:
remove "near the fault"
(and the line below precise
The flux for cells adjacent to the fault, excluding the fault boundary flux.
or something like that?)